src/fiatExchanges/SelectProvider.tsx

```diff
@@ -63,24 +65,45 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
       return
     }
     try {
-      const providers = await fetchProviders({
-        userLocation,
-        walletAddress: account,
-        fiatCurrency: localCurrency,
-        digitalAsset,
-        fiatAmount: route.params.amount.fiat,
-        digitalAssetAmount: route.params.amount.crypto,
-        txType: flow === CICOFlow.CashIn ? 'buy' : 'sell',
-      })
-      const rawLegacyMobileMoneyProviders = await fetchLegacyMobileMoneyProviders()
+      const [
+        fiatConnectQuotes,
+        externalProviders,
+        rawLegacyMobileMoneyProviders,
+      ] = await Promise.all([
+        (async () => {
+          if (fiatConnectEnabled) {
+            const fiatConnectProviders = await getFiatConnectProviders(account)
```

Comment:
I think it would be cleaner if we move this block of code, including the feature flag, `getFiatConnectProviders` and `getFiatConnectQuotes` to `fiatconnect/index` and make a wrapper function called `fetchFiatConnectProviders()`. 

We would end up with 
`await Promise.all([fetchFiatConnectProviders(...params), fetchProviders(...params), fetchLegacyMobileMoneyProviders()`

Reply:
great idea!